apps/pie-microsite/src/tokensMetadata.json

```diff
@@ -189,232 +189,383 @@
         },
         "interactive-inverse": {
-          "category": "interactiveBackgrounds"
+          "category": "interactiveBackgrounds",
+          "description": "Inverse interactive colour used mainly in button and CTA backgrounds",
+          "globalToken": "$color-white"
         },
         "interactive-light": {
-          "category": "interactiveBackgrounds"
+          "category": "interactiveBackgrounds",
+          "description": "Light interactive colour used on dark backgrounds",
+          "globalToken": "$color-white"
         },
         "interactive-form": {
-          "category": "interactiveBackgrounds"
+          "category": "interactiveBackgrounds",
+          "description": "Field background on $background-default",
+          "globalToken": "$color-mozzarella-60"
         },
         "overlay": {
-          "category": "pageBackgrounds"
+          "category": "pageBackgrounds",
+          "description": "Transluscent background overlay / scrim",
+          "globalToken": "$color-black @50%"
+        },
+        "support-neutral": {
+          "category": "feedback",
+          "description": "Neutral; Used for neutral messaging and alerts",
+          "globalToken": "$color-mozzarella-20"
         },
         "support-error": {
-          "category": "feedback"
+          "category": "feedback",
+          "description": "Error; High impact messaging",
+          "globalToken": "$color-red"
         },
-        "support-warning": {
-          "category": "feedback"
+        "support-error-02": {
+          "category": "feedback",
+          "description": "Low contrast version of $support-error",
+          "globalToken": "$color-red-10"
         },
-        "support-positive": {
-          "category": "feedback"
+        "support-warning": {
+          "category": "feedback",
+          "description": "Warning; Moderate impact messaging",
+          "globalToken": "$color-yellow"
         },
-        "support-info": {
-          "category": "feedback"
+        "support-warning-02": {
+          "category": "feedback",
+          "description": "Low contrast version of $support-warning",
+          "globalToken": "$color-yellow-10"
         },
-        "support-neutral": {
-          "category": "feedback"
+        "support-positive": {
+          "category": "feedback",
+          "description": "Success; Positive messaging",
+          "globalToken": "$color-green"
         },
-        "support-error-02": {
-          "category": "feedback"
+        "support-positive-02": {
+          "category": "feedback",
+          "description": "Low contrast version of $support-positive",
+          "globalToken": "$color-green-10"
         },
-        "support-warning-02": {
-          "category": "feedback"
+        "support-positive-inverse": {
+          "category": "feedback",
+          "description": "High contrast version of $support-positive to be used over dark backgrounds",
```

Comment:
The token spreadsheet says to be used over `$container-inverse`, I think that description might be a little bit more helpful

Same for `support-info-inverse` below

Reply:
as mentioned offline I will pick this up with design and invite you along :)